app-backup/deja-dup/deja-dup-42.7.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson eutils gnome2-utils vala xdg
+
+DESCRIPTION="Simple backup tool using duplicity back-end"
+HOMEPAGE="https://gitlab.gnome.org/World/deja-dup"
```

Comment:
Add https://wiki.gnome.org/Apps/DejaDup as the primary homepage, keeping the gitlab page as 2ndary homepage.

Reply:
I assume you refer to https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0104, I removed this.
Is this OK?

Reply:
@oz123 I think he mean something like HOMEPAGE="https://wiki.gnome.org/Apps/DejaDup https://gitlab.gnome.org/World/deja-dup"